OJP/OJP_JourneySupport.xsd

```diff
@@ -63,6 +63,11 @@
 		</xs:annotation>
 		<xs:sequence>
 			<xs:element ref="JourneyRef"/>
+			<xs:element name="PublicCode" type="xs:string" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Descriptive public code of the SERVICE JOURNEY.</xs:documentation>
```

Comment:
Could you please add an example?

Reply:
Not really. However, I harmonized with the NeTEx definition. Perhaps @Aurige has an example.

Reply:
changed